numpyro/infer/util.py

```diff
@@ -416,7 +416,7 @@ def single_chain_init(key):
     return init_params, potential_fn, constrain_fun
 
 
-def predictive(rng, model, posterior_samples, *args, num_samples=None, return_sites=None, **kwargs):
+def predictive(rng, model, posterior_samples, *args, num_samples=None, return_sites=None, parallel=True, **kwargs):
```

Comment:
What do you think about changing this to a class interface like Pyro? I think we should try to get as much of the interface related changes as possible since these are the most difficult to change in later releases.

Reply:
Hmm, I will do it. It will be a good chance to make `predictive` work for `SVI` too.